examples/eval_model.py

```diff
@@ -24,8 +25,16 @@ def main():
     parser = ParlaiParser(True, True)
     parser.add_argument('-n', '--num-examples', default=100000000)
     parser.add_argument('-d', '--display-examples', type='bool', default=False)
+    parser.add_argument('-dbf', '--dict-build-first', type='bool', default=True)
```

Comment:
a dictionary should already be built for a trained model?

Reply:
Yes, I thought it was useful. And the code was copied from `train_model.py`.

Reply:
I think Jason was referring to the fact that a pre-trained model should have already built a dictionary, which was used to train it. You should be loading that dictionary instead of building a new one, because if the new one isn't exactly identical, then the word embeddings stored by the model become meaningless. Thus, we don't necessarily want to build a new one for you here--we should make you go find the one that is saved already.